ipi/engine/normalmodes.py

```diff
@@ -347,15 +347,15 @@ def get_prop_pq(self):
         # Note that the propagator uses mass-scaled momenta.
         for b in range(1, self.nbeads):
             sk = np.sqrt(self.nm_factor[b])
-            square = (self.omegak[b] * dt / 2)**2
-            pqk[b, 0, 0] = (1-square) / (1+square)
-            pqk[b, 1, 1] = (1-square) / (1+square)
-            pqk[b, 0, 1] = (4*square / dt * sk) / (1+square)
-            pqk[b, 1, 0] = dt / sk / (1+square)
-
+            dtomegak = self.omegak[b] * dt / sk
```

Comment:
Sorry, that was an accidental edit of the get_prop_pq(self) when I meant to only change get_cay_prop_pq(self).

I now reverted back to what it used to be before the edit